client/java/src/main/java/io/openlineage/client/Clients.java

```diff
@@ -29,9 +29,16 @@ public static OpenLineageClient newClient(ConfigPathProvider configPathProvider)
     if (isDisabled()) {
       return OpenLineageClient.builder().transport(new NoopTransport()).build();
     }
-    final OpenLineageConfig openLineageConfig =
-        OpenLineageClientUtils.loadOpenLineageConfigYaml(
-            configPathProvider, new TypeReference<OpenLineageConfig>() {});
+    OpenLineageConfig openLineageConfig;
+    try {
+      openLineageConfig =
+          OpenLineageClientUtils.loadOpenLineageConfigYaml(
+              configPathProvider, new TypeReference<OpenLineageConfig>() {});
+    } catch (OpenLineageClientException e) {
+      openLineageConfig =
```

Comment:
Why are we reading conf from env var when exception is thrown?
Shouldn't we read from both places and merge entries, as we already have a mechanism to merge configs.

Reply:
I don't think we want to merge configs from yaml and env vars. The precedence is YAML -> env vars, not a mix of those.